### PR TITLE
Added a new <p>

### PR DESCRIPTION
**Pull request by nach-arch-dev**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
         <h1>Test Page</h1>
         <h2>Test Text</h2>
         <p>This is the description</p>
+        <p>This is additional section</p>
```
Comment (maobarajas) [inline index.html:21]: Why?

**Review by maobarajas — COMMENTED** [review]
Review comments

File: index.html (hunk 1 above)
Comment (nach-arch-dev) [reply index.html:21]: That's why

**Review by maobarajas — APPROVED** [review]
Nice

**Pull request merged**